src/main/java/core/basesyntax/Area.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface Area {
+    public double getArea();

```

Comment:
The 'public' modifier is redundant for interface methods, as they are public by default. You can safely remove it to follow best practices and clean up the code.